### ISSUE TITLE
Performance numbers

### ISSUE THREAD

**Issue by vamsibanda** [nreimers/flax-sentence-embeddings#24]
Hello Nils,

Is there a performance chart of all flax-sentence-embeddings models just like sentence-transformers models (https://www.sbert.net/docs/pretrained_models.html)?

**Comment by nreimers** [comment]
The all-* models are the flax all datasets models v3 / v4. Same for the multi-qa models, these are the same models.

**Comment by vamsibanda** [comment]
Thanks, Nils!

**Comment by stekiri** [comment]
Lots of great resources in this repository, thanks for sharing so much with the community!

I stumbled upon the "Performance" column in the [datasets_list.tsv](https://github.com/nreimers/flax-sentence-embeddings/blob/main/dataset/datasets_list.tsv) and I also saw it on the [corresponding HF dataset](https://huggingface.co/datasets/sentence-transformers/embedding-training-data).

1. How can I interpret the performance numbers per dataset? Does it mean that by using only one particular dataset for training, you obtained the given performance?

2. Given the scores, I assume it corresponds to "Performance Semantic Search (6 Datasets)" referenced on the [Pretrained Models page](https://www.sbert.net/docs/pretrained_models.html) in the sentence-transformers doc. Are the 6 datasets a subset of BEIR or are there other evaluation datasets that you use?

3. It seems that this performance number is also used to determine the weight when training with multiple datasets (see [data_config.json](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/blob/main/data_config.json)), is that correct? Did you also use the size of the dataset in your formula to compute the final weight?

(Sorry for bombarding you with so many questions at once 😅)
